helm/nginx-ingress-controller-app/values.yaml

```diff
@@ -4,7 +4,16 @@
 
 namespace: kube-system
 
-# default provider is bare-metal kvm
+namespace: kube-system
+
+# baseDomain is dynamic environment value, which comes from application catalog configuration
```

Comment:
# baseDomain is a dynamic value, it comes from the cluster values configmap and applies only to Giant Swarm clusters.

Small change to the comments for the dynamic values. WDYT?

Reply:
that sounds better :+1: